operatorcourier/validate.py

```diff
@@ -312,6 +413,31 @@ def _csv_spec_install_validation(self, install):
 
         return valid
 
+    def _csv_descriptors_validation(self, descriptors, crdName):
```

Comment:
Could we also check that the `x-descriptors` (if it exists) field is a list of strings?
Eq: https://github.com/operator-framework/community-operators/blob/master/community-operators/cluster-logging/cluster-logging.v4.1.0.clusterserviceversion.yaml#L275
Not sure if this is a requirement, more like an observation.